### PR TITLE
Bug/1068 render metadata value when compound

### PR DESCRIPTION
**Pull request by kzangeli**
Part of old and closed issue #1068.
Fixed the rendering of compound metadatas when the value is a vector.

An intent to rewrite/simplify that part (before I found the fix) has been kept inside #if 0/#endif.
I don't want to throw away this simplification as I believe we should use it in the near future.

Also, not sure if this fix should have an entry in the CNR ...

### CODE REVIEW

File: src/lib/ngsi/Metadata.cpp [hunk 1]
```diff
@@ -259,13 +259,18 @@ std::string Metadata::render(const std::string& indent, bool comma)
```
Comment (kzangeli) [inline src/lib/ngsi/Metadata.cpp]: Of course.

Fixed in abfb123

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -2,6 +2,7 @@
 - Add: filtering metadata in notifications (#2507)
 - Fix: removed spurious decimals in metadata numbers in NGSIv1 rendering (a follow up of #2176)
 - Fix: crash when creating entity with metadata when subscription is in place with "mq" on a different metadata (#2496)
+- Fix: correct rendering of metadata compound values when a vector
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: I suggest to refer to #1068 here, e.g:

- Fix: correct rendering of metadata compound values when a vector (a bugfix for #1068 functionality)

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: ok

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Fixed in be56f35

**Comment by fgalan** [comment]
LGTM

**Pull request merged**